src/Arguments.hs

```diff
@@ -50,6 +50,9 @@ runOptionsParser strs = execParserPure prefs optionsParser strs
       { prefMultiSuffix = "SPARK"  -- metavar suffix for multiple options
       , prefDisambiguate = True    -- automatically disambiguate abbreviations (default: False)
       , prefShowHelpOnError = True -- always show help text on parse errors (default: False)
+#if MIN_VERSION_base(4,9,1)
```

Comment:
I think you want a `MIN_VERSION` of `optparse-applicative` instead.

Reply:
oh! I didn't know you could do that. I'll fix this in a bit.